shared/login/reset/confirm.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+
+const todo = () => console.log('todo')
+
+type Props = {
+  hasWallet: boolean
+}
+
+const ConfirmReset = (props: Props) => {
+  const [check1, setCheck1] = React.useState(false)
```

Comment:
i think this is too disconnected from the content. 
This should either be a map like `{wallet: false, ...}` or at least `checkWallet` etc